tests/services/test_file_freezer.py

```
@@ -152,23 +150,23 @@ def test_freeze_file_already_in_cold(self, db_session, file_inventory_factory, s
+
         success, error, cold_path = FileFreezer.freeze_file(
             inv, monitored_path, storage_location, db=db_session
         )
-        
+
         assert success is False
         assert "not in hot storage" in error.lower()
 
     def test_freeze_file_not_found_on_disk(self, db_session, file_inventory_factory, storage_location):
         """Test that freezing fails if file is missing from hot storage."""
         inv = file_inventory_factory(path="/tmp/hot/missing.txt", storage_type=StorageType.HOT)
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/hot/missing.txt"

(S108)